doc/multi_process_backend.md

```diff
@@ -0,0 +1,33 @@
+## Multi Process Backend
+
+It is attempt to have backend consisting of several dbus services that communicates between.
+The goal of attempt is to verify that having separate services can speed up probing phase and maybe even slightly installation one.
```

Comment:
Another important goal is to be responsive (via D-Bus) even if d-installer is busy. For instance, while reading metadata from repositories or installing packages, the D-Bus interface does not respond. We would like to avoid such a problem (at least with long-term operations).